test/e2e/lib/driver-helper.js

```diff
@@ -567,3 +567,11 @@ export async function waitTillTextPresent( driver, selector, text, waitOverride
 		`Timed out waiting for element with ${ selector.using } of '${ selector.value }' to be present and displayed with text '${ text }'`
 	);
 }
+
+export async function waitTillAbleToSwitchToFrame( driver, selector, text, waitOverride ) {
+	const timeoutWait = waitOverride ? waitOverride : explicitWaitMS;
+	const timeoutText = text
+		? text
+		: `Timed out wiating to be able to switch to IFrame with ${ selector.using } of ${ selector.value }`;
+	return driver.wait( until.ableToSwitchToFrame( selector ), timeoutWait, timeoutText );
+}
```

Comment:
Recently, I've been doing a bit of refactoring of the driver helpers and discovered a few ways we can streamline and improve this one at the same time. Here's my suggestion (based on the recently refactored `waitUntilElementWithTextLocated` helper):

```suggestion
/**
 * Waits until the input driver is able to switch to the designated frame.
 * Upon successful resolution, the driver will be left focused on the new frame.
 *
 * @param {WebDriver} driver The parent WebDriver instance
 * @param {By} locator The element's locator
 * @param {number} [timeout=explicitWaitMS] The timeout in milliseconds
 * @returns {Promise<boolean>} A promise that will resolve with true if the switch was succesfull
 */
export function waitUntilAbleToSwitchToFrame( driver, locator, timeout = explicitWaitMS ) {
	return driver.wait( until.ableToSwitchToFrame( locator ), timeout );
}
```

#### Here's what's changed:
- Added some docs for clarity
- Used `*Until*` instead of `*Till*` to represent the actual module used
- Used `locator` instead of `selector` as it's the correct name for the `By` instance (see i.e. [this source](https://github.com/SeleniumHQ/selenium/blob/trunk/javascript/node/selenium-webdriver/lib/by.js#L146-L155))
- Assigned `explicitWaitMS` directly as the default `timeout` arg value
- Removed obsolete `async` keyword
- Removed the custom error message because [it's being composed internally](https://github.com/SeleniumHQ/selenium/blob/trunk/javascript/node/selenium-webdriver/lib/until.js#L85) by the used `until.ableToSwitchToFrame` helper. We also shouldn't need the customizable error message (the `text` arg).